app/services/store.js

```diff
@@ -60,4 +61,17 @@ export default class ExtendedStoreService extends Store {
       'filter[:uri:]': uri,
     });
   }
+
+  // TODO: Move this to a concept-service where we bundle logic for getting
+  // all concepts of a scheme/specific concepts? Might cut down some of the
+  // repetition across the codebase.
+  queryConceptsForConceptScheme(conceptSchemeUri) {
```

Comment:
Probably a good idea to move this to a separate service, since it's becoming a bit too 'model-specific' to be an extension of the generic ember-data store. Another option is a util method that gets the store as argument, but not sure if I like that more.

Can you also use `this.queryAll` instead of `this.query`? This way we're sure we always load the complete codelist. I assume `PAGE_SIZE.CODE_LISTS` can be removed afterwards.

Reply:
I had some issues with `queryAll`, there is an option to pass in the page size but when you do, the outgoing query contains `page:size` twice (in different formats)